packages/tools/fetch-tool/src/fluidFetchMessages.ts

```diff
@@ -140,16 +191,28 @@ async function* saveOps(
     // Split into 100K ops
     const chunk = 100 * 1000;
 
+    let sequencedMessages: ISequencedDocumentMessage[] = [];
+    // current sequence number to read from
+    let curr = 1;
```

Comment:
fyi, curr is the first op we want to write to that file and not the op we want to fetch. What we fetch is already decided in above loadAllSequencedops api. Here we are just fetching from "gen" without specifying from where wwe want to fetch.